### PR TITLE
Enhancement Community Member Profile Social Links #4367

### LINKED ISSUE #4367: Enhancement Community Member Profile Social Links
### Desired Behavior

Add Meshery Cloud public profile link of the member next to LinkedIn icon.

1. Add Meshery Cloud logo next to the LinkedIn icon.
2. Add meshery cloud UUID as input in "member-profile-template".

https://layer5.io/community/members/aadhitya-amarendiran

### Screenshots / Mockups

Screenshot 2023-06-09 at 2 32 30 PM

### Acceptance Tests

---
<h2>Contributor Resources and <a href="https://layer5.io/community/handbook">Handbook</a></h2>

The layer5.io website uses Gatsby, React, and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
- 📚 See [contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md).
- 🎨 Wireframes and designs for Layer5 site in [Figma](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs).
- 🙋🏾🙋🏼 Questions: [Discussion Forum](https://discuss.layer5.io) and [Community Slack](https://slack.layer5.io).

Join the Layer5 Community by submitting your [community member form](https://layer5.io/newcomer).

### PR DESCRIPTION
**Pull request by Manish254252**
**Description**

This PR fixes #4367 

**Notes for Reviewers**

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

File: src/sections/Community/Handbook/Handbook.style.js [hunk 1]
```diff
@@ -338,13 +338,14 @@ export const HandbookWrapper = styled.div`
...
     .channels-img {
-      width: 40px;
+      width: 30px;
       height: 30px;
     }
 
     .channels-para {
       display: flex;
       align-items: flex-end;
+
```
Comment (samyakshah3008) [inline src/sections/Community/Handbook/Handbook.style.js:348]: Can we review this unwanted empty line?

File: src/sections/Community/Member-single/index.js [hunk 2]
```diff
@@ -251,6 +252,13 @@ const MemberSingle = ({ frontmatter }) => {
                       </a>
                     </li>
                   )}
+                  {
+                    <li>
+                      <a href="https://layer5.io/cloud-native-management/meshery">
```
Comment (samyakshah3008) [inline src/sections/Community/Member-single/index.js:257]: Can we add `target="_blank"` here?

File: src/sections/Home/Partners-home/partnerSection.style.js [hunk 3]
```diff
@@ -1,103 +1,94 @@
...
+  .partner-image {
+    margin-left: 1.5rem;
+    margin-right: 1.5rem;
+  }
+  .partner-slider {
+    display: flex;
+    justify-content: center;
+    flex-wrap: nowrap;
+    @media (max-width: 1400px) {
+      flex-wrap: wrap;
```
Comment (samyakshah3008) [inline src/sections/Home/Partners-home/partnerSection.style.js:23]: Can we add `screen` as well? 
Updated code can be: `@media screen and (max-width: 1400px)` ?

File: src/sections/Home/Partners-home/partnerSection.style.js [hunk 4]
```diff
@@ -1,103 +1,94 @@
...
 
-        #VMware:hover
-        {
-        
-           filter:brightness(10%);
-        
-
-         }
+  a.partner-card {
+    @media (max-width: 1400px) {
```
Comment (samyakshah3008) [inline src/sections/Home/Partners-home/partnerSection.style.js:39]: I am not able to understand why you have written two @media queries for the same max-width i.e 1400px. 
You can write under one media query for same width. 
Or please correct me where I am wrong.

File: static/images/meshery-logo-light.webp
Comment (samyakshah3008) [inline static/images/meshery-logo-light.webp:1]: Can we prefer `svg` over `webp`?

**Review by samyakshah3008 — COMMENTED** [review]
Great work @Manish254252 💪  

I have added few single line comments at few pieces of code, please check. 
Rest looks good to me! 🚀

**Comment by vishalvivekm** [comment]
@Manish254252 Let's discuss this on the Websites call. I'll add this as an agenda item in the [meeting minutes](https://docs.google.com/document/d/1XczAHXVe2FIWPqiF57ospJ43zw5cZQ7ui8mn39v5EvA/edit#). // @samyakshah3008

File: src/sections/Home/Partners-home/partnerSection.style.js (hunk 4 above)
Comment (Manish254252) [reply src/sections/Home/Partners-home/partnerSection.style.js:39]: Actually it was like that when I clone it so I am not sure why it is like that.

File: src/sections/Home/Partners-home/partnerSection.style.js (hunk 3 above)
Comment (Manish254252) [reply src/sections/Home/Partners-home/partnerSection.style.js:23]: sure I will make changes .

File: src/sections/Community/Member-single/index.js (hunk 2 above)
Comment (Manish254252) [reply src/sections/Community/Member-single/index.js:257]: Done.

File: src/sections/Community/Handbook/Handbook.style.js (hunk 1 above)
Comment (Manish254252) [reply src/sections/Community/Handbook/Handbook.style.js:348]: Done.

**Comment by samyakshah3008** [comment]
You're most welcome, Manish!

**Comment by Shivam-AfA** [comment]
@Manish254252 We have merge conflicts here. Please resolve them.

**Comment by Shivam-AfA** [comment]
@Manish254252 There are a lot of unnecessary changes in the PR. Please do not use any formatter while pushing commits. Use `npm run lint` before comitting.

**Comment by Shivam-AfA** [comment]
Seems like they are still not resolved @Manish254252

**Pull request closed without merging**